reconcile/utils/terrascript_aws_client.py

```diff
@@ -1181,6 +1185,64 @@ def populate_vpc_peerings(self, desired_state):
                     tf_resource = aws_route(route_identifier, **values)
                     self.add_resource(infra_account_name, tf_resource)
 
+    def populate_vpc_requests(self, vpc_requests: Iterable[VPCRequest]) -> None:
+        for request in vpc_requests:
+            # The default values here come from infra repo's module configuration
+            values = {
+                "source": "terraform-aws-modules/vpc/aws",
+                "version": "5.7.1",
+                "name": request.identifier,
+                "cidr": request.cidr_block.network_address,
+                "private_subnet_tags": {"kubernetes.io/role/internal-elb": "1"},
+                "public_subnet_tags": {"kubernetes.io/role/elb": "1"},
+                "create_database_subnet_group": False,
+                "enable_dns_hostnames": True,
```

Comment:
I suppose we have no reason to believe we might want to tweak those in the future and hardcoding them is fine and will fit all of our needs? We can also parametrize them later as needed. Just double-checking...

Reply:
Yeah, trying to prevent early abstractions.